module3/assessments/week5-1422.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Week 5 Assessment
+---
+
+[Follow the instructions on this repo!](https://github.com/turingschool-examples/Launch_6490)
```

Comment:
This is the link to the Week 3 assessment repo

Reply:
uggghhh - numbers are hard.  It is updated with the correct link!